nvt-sync.yml

```diff
@@ -7,7 +7,7 @@ services:
       - openvas-var-lib:/var/lib/openvas
       - run-redis:/var/run/redis
       - run-ospd:/var/run/ospd
-    command: greenbone-nvt-sync
+    command: greenbone-nvt-sync --rsync
```

Comment:
```suggestion
    command: ["greenbone-nvt-sync", "--rsync"]
```

Reply:
makes sense - have tested and agree :) thanks for correcting me on this! 😃